sentry_sdk/tracing.py

```diff
@@ -1267,12 +1265,18 @@ def __exit__(self, ty, value, tb):
         # XXX set status to error if unset and an exception occurred?
         context.detach(self._ctx_token)
 
+    def _get_attribute(self, name):
+        # type: (str) -> Optional[Any]
+        if not isinstance(self._otel_span, ReadableSpan):
+            return None
```

Comment:
We can have a decorator at some point that does this, we could reuse that everywhere for this is-recording-span check. Don't have to do it now, just a note for later when we're cleaning stuff up